Index: src/main/java/ua/lov/mathematic/Discr.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/ua/lov/mathematic/Discr.java	(revision 22213929ca7a8e3b9c3f3da9df9703d393276588)
+++ src/main/java/ua/lov/mathematic/Discr.java	(date 1634581918400)
@@ -1,4 +1,4 @@
-package mathematic;
+package ua.lov.mathematic;
 
 import java.util.ArrayList;
 import java.util.HashMap;
Index: .gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- .gitignore	(revision 22213929ca7a8e3b9c3f3da9df9703d393276588)
+++ .gitignore	(date 1634658714020)
@@ -46,3 +46,4 @@
 parsing.txt
 /src/main/java/ua/lov/dost/note/
 /src/main/java/ua/lov/dost/note/
+parsing.txt
Index: src/main/java/ua/lov/mathematic/StreamlongLong.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/ua/lov/mathematic/StreamlongLong.java	(revision 22213929ca7a8e3b9c3f3da9df9703d393276588)
+++ src/main/java/ua/lov/mathematic/StreamlongLong.java	(date 1634626873299)
@@ -1,4 +1,4 @@
-package mathematic;
+package ua.lov.mathematic;
 
 import java.util.Arrays;
 import java.util.Objects;
Index: src/main/java/ua/lov/mathematic/Main.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/ua/lov/mathematic/Main.java	(revision 22213929ca7a8e3b9c3f3da9df9703d393276588)
+++ src/main/java/ua/lov/mathematic/Main.java	(date 1634581923416)
@@ -1,4 +1,4 @@
-package mathematic;
+package ua.lov.mathematic;
 
 import java.util.Arrays;
 import java.util.Objects;
@@ -6,7 +6,7 @@
 public class Main {
     public static void main(String[] args) {
         Discr discr = new Discr();
-  //      System.out.println(discr.quare(1, -1, -90));
+        System.out.println(discr.quare(3, -7, 2));
   //      System.out.println(discr.quare(1, -1, 1));
   //      System.out.println(discr.quare(1, 12, 17));
   //      System.out.println(discr.quare(1, 4, -15));
Index: src/main/java/ua/lov/dost/DostavkApp.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/ua/lov/dost/DostavkApp.java	(revision 22213929ca7a8e3b9c3f3da9df9703d393276588)
+++ src/main/java/ua/lov/dost/DostavkApp.java	(date 1634406408636)
@@ -29,8 +29,8 @@
         String str = "";
         try {
             while ((str = br.readLine()) != null) {
-                String lowcase = "("+str.toLowerCase()+")\n";
-             //   String lowcase = "'"+str.toLowerCase()+"'\n";
+            //    String lowcase = "("+str.toLowerCase()+")\n";
+                String lowcase = "'"+str.toLowerCase()+"'\n";
 
                 stringBuilder.append(",").append(lowcase);
             }
